server/src/main/java/org/elasticsearch/cluster/coordination/indicators/NoEligibleMasterNodesIndicator.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.cluster.coordination.indicators;
+
+import org.elasticsearch.cluster.node.DiscoveryNode;
+import org.elasticsearch.cluster.node.DiscoveryNodeRole;
+import org.elasticsearch.cluster.service.ClusterService;
+import org.elasticsearch.health.HealthIndicatorResult;
+import org.elasticsearch.health.HealthIndicatorService;
+import org.elasticsearch.health.HealthStatus;
+
+public class NoEligibleMasterNodesIndicator implements HealthIndicatorService {
+
+    public static final String NO_ELIGIBLE_MASTERS = "no_eligible_masters";
```

Comment:
`no_eligible_masters: green` <- no_eligible_masters is bad, but green is good. I am confused.
May be has_aligible_masters?

Here and in another pr: https://github.com/elastic/elasticsearch/pull/83750/files#r803384855

Reply:
Thank you! I totally agree that we should avoid double negatives.